doc/Comments.xml

```diff
@@ -297,6 +319,25 @@ Order(S5);
 ]]></Listing>
 </Subsection>
 
+
+<Subsection Label="@Listing">
+<Index Key="@Listing"><C>@Listing, @BeginListing, and @EndListing</C></Index>
+<Heading>@BeginListing <A>tested</A> and @EndListing</Heading>
+@BeginListing inserts an example into the manual. Opposed to @Example they have
+the same Syntax as GAPDoc examples and each line has to begin with a GAPDoc comment.
+@EndListing ends the example block.
+If <A>tested</A> is <C>false</C>, the example will not be tested if the manual examples are tested.
```

Comment:
`tested` to me "feels" wrong... I"d prefer "test" or "addToTests" or "includeInTests" or so... Hrm...

Also, I would make this optional -- by default, all listings / examples are tested, but I can mark some of them so that they will be not tested.

Indeed it might be easier if you did not allow a boolean here, but rather `@Listing` and `@Listing notest` or so.